sdk/webpubsub/azure-messaging-webpubsub/CHANGELOG.md

```diff
@@ -4,11 +4,9 @@
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added method `removeConnectionFromAllGroupsWithResponse` in `WebPubSubServiceClient` and `WebPubSubServiceAsyncClient` to remove the connection from all the groups it is in.
```

Comment:
Shall we also update the Readme.md samples to include these new features, especially the initial group and the send with filter? A similar approach in c# could be found here https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/webpubsub/Azure.Messaging.WebPubSub#broadcast-messages-to-clients-using-filter

Reply:
Talked with @srnagar and @weidongxu-microsoft, we are not inclined to add this filter class for WPS, the reasons are:
1. Tt's mostly a DPG client with no models (except for some enums). 
2. This is OData filter which is common across many other services.
3. if we do have to add it, we should consider adding it in azure-core.

What we will do now is to add sample with filter to let user know how to use filter.

Reply:
samples added in this commit: https://github.com/Azure/azure-sdk-for-java/pull/32125/commits/5fe90e73bfd59124dbdca9670234618fdff0c06d

Reply:
I am cool with that